src/projectscene/view/trackspanel/trackcontextmenumodel.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+* Audacity: A Digital Audio Editor
+*/
+#include "trackcontextmenumodel.h"
+
+using namespace au::projectscene;
+using namespace muse::uicomponents;
+using namespace muse::actions;
+
+void TrackContextMenuModel::load()
+{
+    AbstractMenuModel::load();
+
+    auto makeItemWithArg = [this](const ActionCode& actionCode) {
+        MenuItem* item = makeMenuItem(actionCode);
+        item->setArgs(ActionData::make_arg1<trackedit::TrackId>(m_trackId));
+        return item;
+    };
+
+    MenuItem* rename = new MenuItem(this);
+    rename->setTitle({ "Rename track", "Rename" });
```

Comment:
Here you need to explicitly specify TranslatableString
Otherwise the utility for generating the file for translation will not process these lines (there is just a parser by keywords, TranslatableString is a keyword)

Reply:
I see, thanks